config/samples/banzaicloud_v1beta1_kafkacluster.yaml

```
@@ -316,8 +316,11 @@ spec:
     # sslSecrets contains information about ssl related kubernetes secrets if one of the
     # listener setting type set to ssl these fields must be populated too.
     sslSecrets:
-      # tlsSecretName should contain all ssl certs required by kafka including: caCert, caKey, clientCert, clientKey
-      # serverCert, serverKey, peerCert, peerKey
+      # when create is false then the user has to provide the TLS certificate(s) and private key in PEM format in the refererenced secret below.
+      # this certificate and private key is used by cert-manager to issue client and server certificates. The client certificate is used by the Koperator , Cruise Control, 
+      # and Cruise Control Metrics Reporter to communicate Kafka brokers using listener with SSL enabled.
+      # The server certificate is used by SSL kafka listeners.
```

Not sure if this sentence was meant to say the following:
```suggestion
      # The server certificate is used for SSL authentication through Kafka listeners.
```

---

It was not. This is for kafka listeners.

---

But the way that this sentence is worded is a bit confusing, what does it really mean that the certificate is used by "SSL kafka listeners"? Is "SSL kafka listeners" a term that I am not aware of?

---

I think it means listeners using SSL (`ssl` and `sasl_ssl` listener types), or that would be my interpretation, is that correct @bartam1 ?